recipes/unicloud/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "unicloud" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Serapieum-of-alex/unicloud/archive/{{ version }}.tar.gz
+  sha256: ddac4a0573c2b86c519ae02fcc8ed1d0390c7619d277756f1db26d66c522b1d6
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
```

Comment:
Is this correct? If so, this package cannot be noarch.

Reply:
removed